src/app/shared/ui/circular-button/circular-button.component.ts

```diff
@@ -0,0 +1,31 @@
+import { CommonModule } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+} from '@angular/core';
+import { MatIconModule } from '@angular/material/icon';
+
+type IconOptions = 'done' | 'close' | 'arrow_back' | 'arrow_forward';
+type sizeOption = 'sm' | 'md' | 'lg' | 'xl';
+
+@Component({
+  selector: 'app-circular-button',
+  standalone: true,
+  imports: [MatIconModule, CommonModule],
+  templateUrl: 'circular-button.html',
+  styleUrls: ['./circular-button.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CircularButtonComponent {
+  @Input() icon?: IconOptions;
+  @Input() size?: sizeOption;
```

Comment:
nie ustawiamy domyślnych wartości?

Reply:
i tak i nie. Dla Inputa icon jeżeli nie dajemy wartości to ma przypisany text "Dołącz", który będzie wykorzystany do kart. Okrągłych buttonów z innym textem w makietach nie mamy dlatego tylko tez nie robilem inputa dla text, chociaz w przyszłości może pojawić się taka potrzeba wiec dodam taka możliwość z domyślnym textem "Dołącz". Do Inputa size faktycznie powinienem dodać wartość domyślna.